code/Test_definitions/geofencing-subscriptions.feature

```diff
@@ -213,6 +213,15 @@ Feature: Camara Geofencing Subscriptions API, vwip - Operations on subscriptions
     And the response property "$.code" is "INVALID_TOKEN" or "INVALID_ARGUMENT"
     And the response property "$.message" contains a user friendly text
 
+  @geofencing_subscription_17_invalid_sink
```

Comment:
In the artifact the model is [ @<xxx>_subscription_26_invalid_url](https://github.com/camaraproject/Commonalities/blob/main/artifacts/camara-cloudevents/event-subscription-template.feature#L204)